lib/gateways/notification/govNotify.js

```diff
@@ -0,0 +1,28 @@
+module.exports = ({ client, log }) => {
+  function send(dropbox, requestedBy, requestedByEmail) {
+    try {
+      log.info('Sending email notification to', {
+        requestedBy
+      });
+
+      const personalisation = {
+        name: requestedBy,
+        residentName: dropbox.customerName,
+        description: dropbox.description,
+        linkToDropbox: `${process.env.URL_PREFIX}/dropboxes/${dropbox.id}/view`
+      };
+
+      return client
+        .sendEmail(process.env.GOV_NOTIFY_TEMPLATE_ID, requestedByEmail, {
+          personalisation,
+          reference: null
```

Comment:
What is this needed for?

Reply:
It says it's required in documentation
_A unique identifier you create. This reference identifies a single unique notification or a batch of notifications. It must not contain any personal information such as name or postal address. If you do not have a reference, you must pass in an empty string or null._